File: Python/2021/1stOct_nekkantimadhu.py
def FibonacciNumbers(n):
 
    f1 = 1
    f2 = 1
    if (n < 1):
        return
    print(f1 , end = " ")
    for x in range(1, n):
        if(f2 % 5 == 0):
            print(0 , end = " ")
        else:
            print(f2 , end = " ")
        sum = f1 + f2
        f1 = f2
        f2 = sum

# Driven code
while(True):
    n = int(input())
    FibonacciNumbers(n)
    print()

File: JAVA/2021/1stOct_nekkantimadhu.java
import java.util.*;
public class Main {
    static void FibonacciNumbers(int n)
    {
        int f1 = 1, f2 = 1, i;
 
        if (n < 1)
            return;
        System.out.print(f1 + " ");
        for (i = 1; i < n; i++){
            if(f2 % 5 == 0){
                System.out.print(0 + " ");
            }
            else{
                System.out.print(f2 + " ");
            }
            
            int sum = f1 + f2;
            f1 = f2;
            f2 = sum;
        }
    }
 
    // Driver Code
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        while(true){
            int n = sc.nextInt();
            FibonacciNumbers(n);
            System.out.println("");
        }
        
    }
}